tests/test-values.yaml

```diff
@@ -59,9 +59,9 @@ hostAliases: []
 ## Bitnami Kubewatch image version
 ## ref: https://hub.docker.com/r/bitnami/kubewatch/tags/
 ##
-## @param image.registry Kubewatch image registry
-## @param image.repository Kubewatch image name
-## @param image.tag Kubewatch image tag
+## @param image.registry [default: REGISTRY_NAME] Kubewatch image registry
+## @param image.repository [default: REPOSITORY_NAME/kubewatch] Kubewatch image name
+## @skip image.tag Kubewatch image tag
 ## @param image.pullPolicy Kubewatch image tag
```

Comment:
Aaaargh! please accept this change I was getting crazy thinking that the `skip` tag was doing something wrong
```suggestion
## @param image.pullPolicy Kubewatch image pull policy
```